config/samples/sw.kogito_v1alpha08_kogitoserverlessplatformWithBaseImage.yaml

```diff
@@ -1,10 +0,0 @@
-apiVersion: sw.kogito.kie.org/v1alpha08
```

Comment:
didn't we discuss the we wouldn't change the config/samples?

Reply:
We have agreed the proposal of @davidesalerno to use the needed yaml files and create the structs with the few fields needed in the tests instead to have one yaml file for each test, this file is replaced by the platform plus the field with the base image

Reply:
The `config/samples` directory is used by the OLM/Operator SDK to compose the CSV. Meaning that users will these examples on the OpenShift console. 

Bear that in mind and keep only the simplest ones. I think this is the proposal, right?

Reply:
So we should keep here only the ones that will be in the CSV, **one very simple example per custom resource**.

Reply:
For example, see here: https://operatorhub.io/operator/nexus-operator-m88i 
Click on "View Example".

That's the purpose of this directory.

Reply:
@ricardozanini  Yep the file used by kustomization 
https://github.com/kiegroup/kogito-serverless-operator/blob/main/config/samples/kustomization.yaml
are only: 
sw.kogito_v1alpha08_kogitoserverlessworkflow.yaml
sw.kogito_v1alpha08_kogitoserverlessplatform.yaml 

and in the folder in the PR we leave only these two and sw.kogito_v1alpha08_kogitoserverlessworkflow_devmodeWithConfigMapAndExternalResource.yaml with a complete example, but we can move it in the docs folder

Reply:
I think we should keep the config/samples as it is, not remove anything, maybe the similar examples, e.g. greeting, and greeting-ITA.
it was my understand from what we discussed on Tuesday.

Reply:
AFAIK, only the examples added in the kustomize files will be added to the CSV, we can keep then around that it will not hurt anything and can serve as examples with more complex workflows.